app/src/main/java/com/besome/sketch/editor/manage/library/admob/ManageAdmobActivity.java

```diff
@@ -201,6 +209,7 @@ public void onCreate(Bundle savedInstanceState) {
         A = new DB(getApplicationContext(), "P1");
         admobLibraryBean = getIntent().getParcelableExtra("admob");
         ((TextView) findViewById(R.id.tv_enable)).setText(Helper.getResString(R.string.design_library_settings_title_enabled));
+        ((TextView) findViewById(R.id.tv_title_app_id)).setText(Helper.getResString(R.string.design_library_admob_title_app_id));
```

Comment:
Since these strings aren't translated so you shouldn't use `Helper#getResString(int)`.
You can use that string in XML.

Reply:
So what you mean is I'll just use setText(R.string.string_name)?